### PR TITLE
Update tracker and Ad's explanation.

### PR DESCRIPTION
**Pull request by Miiiiaaooooo**

### CODE REVIEW

**Review by Kiloutre — DISMISSED** [review]

File: translations/en-us.all.json [hunk 1]
```diff
@@ -1893,7 +1893,9 @@
   },
   {
     "id": "faq_ads_explanation",
-    "translation": "[lengthy explanation as to why we have ads"
+    "translation": "This is not a decision we make lightly, and we have thoroughly considered the objections raised by all of you, but ultimately we feel this is the only sustainable solution to fund Pantsu and ensure its continued operation. \
```
Comment (akuma06) [inline translations/en-us.all.json]: Use `<br>` html tag instead. JSON format doesn't allow line-breaks.

**Review by akuma06 — CHANGES_REQUESTED** [review]

File: translations/en-us.all.json (hunk 1 above)
Comment (Kiloutre) [reply translations/en-us.all.json]: I'm still keeping the \ because having an extremely long line isn't really good for translation purposes

Comment (akuma06) [inline translations/en-us.all.json]: In JSON format, you can't have line-breaks in string. You have to put every string in one line (only). That's how the format works. And that's why your travis fails

**Review by akuma06 — DISMISSED** [review]

File: translations/en-us.all.json (hunk 1 above)
Comment (akuma06) [reply translations/en-us.all.json]: https://stackoverflow.com/questions/2392766/multiline-strings-in-json

**Review by Kiloutre — APPROVED** [review]

File: translations/en-us.all.json (hunk 1 above)
Comment (yiiTT) [inline translations/en-us.all.json]: doko.moe > yw0.xyz has already been done across all translations FYI

**Comment by yiiTT** [comment]
doko.moe > yw0.xyz has already been done across all translations FYI

**Pull request merged**